functions.php

```diff
@@ -78,3 +78,124 @@ function displayGuitars(array $guitarsArray):string {
     }
     return $result;
     }
+
+function checkBrands(\PDO $database):bool {
```

Comment:
For the future - DEFINITELY NOT NOW - a transaction might be useful here as all the queries seem to be needed together

Reply:
Or take the fieldname and table as a parameter into a checkForAThing function